iree/compiler/Codegen/SPIRV/SPIRVTile.cpp

```diff
@@ -31,6 +31,8 @@
 namespace mlir {
 namespace iree_compiler {
 
+static constexpr int kNumGPUDims = 3;
```

Comment:
Is this even used anywhere?

Reply:
Yes used later. Now changed to getting the definition from GPUUtils.h.